spec/unit/shoulda/matchers/action_controller/permit_matcher_spec.rb

```diff
@@ -283,10 +377,7 @@
     Class.new(StandardError)
   end
 
-  def define_controller_for_resource_with_strong_parameters(
-    options = {},
-    &action_body
-  )
+  def define_controller_with_strong_parameters(options = {}, &action_body)
```

Comment:
Assignment Branch Condition size for define_controller_with_strong_parameters is too high. [20.98/15]